docs/android/v2/changelog/release-notes.mdx

```diff
@@ -3,6 +3,16 @@ title: Release Notes
 nav: 15
 description: Release Notes for 100ms Android SDK
 ---
+## v2.5.6 - 2023-01-11
+### Added
+* Send `requested_by` in acceptChangeRole message
+* Added `CameraControls` interface to `HMSLocalVideoTrack` - Use this interface to control the settings of native camera
+* Added Utils method to know about the properties of vp8 codecs present in the device
```

Comment:
shouldnt we have link to the android docs page which talks about this util ?

Reply:
Since its not very useful right now, so not adding. 

But a link to the docs of Cmaera controls would be useful

Reply:
Actually removing the utils method from releaee notes for now and adding link toCameraControls